routes/major.js

```diff
@@ -21,4 +23,13 @@ router.post("/api/majors", async (req, res) => {
     .catch((err) => errorHandler(res, 400, err));
 });
 
+// Once we add all majors to the majors.js file, we can use this code to add all majors to production DB
+router.post("/api/majors/all", async (req, res) => {
```

Comment:
let's talk about this lol

Reply:
Yeah that was just a quick guess (not tested) assuming we continue updating AllMajors in majors.js